websites/api-docs-smoke-test/package.json

```diff
@@ -2,6 +2,7 @@
   "name": "@commercetools-website/api-docs-smoke-test",
   "version": "6.0.21",
   "private": true,
+  "browserslist": [">0.25%", "not dead and supports es6-module"],
```

Comment:
I wondered, too whether we can do that already.    I think it's OK because the site degrades gradually if the JS fails (not completely but at least you can move around the content if you found it through google). 

We'd have to apply it manually to the repositories using the kit, right?

Reply:
> I wondered, too whether we can do that already.

I think we should be good. I checked the supported browsers and these are all old versions https://browserslist.dev/?q=PjAuMjUlLCBub3QgZGVhZCBhbmQgc3VwcG9ydHMgZXM2LW1vZHVsZQ%3D%3D

> We'd have to apply it manually to the repositories using the kit, right?

I haven't looked much into it but yes I suppose so.

Reply:
I checked the readme, it can live in the package.json of a parent directory, too or in a separate config file in a parent directory:  https://github.com/browserslist/browserslist#queries   

That would be less maintenance than having it in every website (esp in the commercetools-docs repo which is getty pretty big now)